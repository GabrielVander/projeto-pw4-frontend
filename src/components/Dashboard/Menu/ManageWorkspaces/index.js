import React from "react";
import Card from "react-bootstrap/Card";
import {BsFolder} from "react-icons/bs";
import { motion } from "framer-motion"

function ManageWorkspaces() {
    return (
        <>
            <motion.div
                whileHover={{ scale: 1.1 }}
                whileTap={{ scale: 0.9 }}
            >
                <Card bg='danger'>
                    <Card.Header>Manage workspaces</Card.Header>
                    <Card.Body>
                        <BsFolder size={50}/>
                    </Card.Body>
                </Card>
            </motion.div>
        </>
    );
}

export default ManageWorkspaces;