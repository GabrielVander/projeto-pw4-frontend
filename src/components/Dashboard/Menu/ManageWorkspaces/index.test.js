import React from 'react';
import { render } from '@testing-library/react';
import ManageWorkspaces from './index';

describe('Component renders properly', () => {
	test('Matches snapshot', () => {
		const component = render(<ManageWorkspaces/>);
        
		expect(component).toBeDefined();
		expect(component).toMatchSnapshot();
	});
});