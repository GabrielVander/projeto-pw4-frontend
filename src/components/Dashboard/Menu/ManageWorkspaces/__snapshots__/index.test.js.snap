// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component renders properly Matches snapshot 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div>
        <div
          class="card bg-danger"
        >
          <div
            class="card-header"
          >
            Manage workspaces
          </div>
          <div
            class="card-body"
          >
            <svg
              fill="currentColor"
              height="50"
              size="50"
              stroke="currentColor"
              stroke-width="0"
              viewBox="0 0 16 16"
              width="50"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M9.828 4a3 3 0 01-2.12-.879l-.83-.828A1 1 0 006.173 2H2.5a1 1 0 00-1 .981L1.546 4h-1L.5 3a2 2 0 012-2h3.672a2 2 0 011.414.586l.828.828A2 2 0 009.828 3v1z"
              />
              <path
                clip-rule="evenodd"
                d="M13.81 4H2.19a1 1 0 00-.996 1.09l.637 7a1 1 0 00.995.91h10.348a1 1 0 00.995-.91l.637-7A1 1 0 0013.81 4zM2.19 3A2 2 0 00.198 5.181l.637 7A2 2 0 002.826 14h10.348a2 2 0 001.991-1.819l.637-7A2 2 0 0013.81 3H2.19z"
                fill-rule="evenodd"
              />
            </svg>
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div>
      <div
        class="card bg-danger"
      >
        <div
          class="card-header"
        >
          Manage workspaces
        </div>
        <div
          class="card-body"
        >
          <svg
            fill="currentColor"
            height="50"
            size="50"
            stroke="currentColor"
            stroke-width="0"
            viewBox="0 0 16 16"
            width="50"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M9.828 4a3 3 0 01-2.12-.879l-.83-.828A1 1 0 006.173 2H2.5a1 1 0 00-1 .981L1.546 4h-1L.5 3a2 2 0 012-2h3.672a2 2 0 011.414.586l.828.828A2 2 0 009.828 3v1z"
            />
            <path
              clip-rule="evenodd"
              d="M13.81 4H2.19a1 1 0 00-.996 1.09l.637 7a1 1 0 00.995.91h10.348a1 1 0 00.995-.91l.637-7A1 1 0 0013.81 4zM2.19 3A2 2 0 00.198 5.181l.637 7A2 2 0 002.826 14h10.348a2 2 0 001.991-1.819l.637-7A2 2 0 0013.81 3H2.19z"
              fill-rule="evenodd"
            />
          </svg>
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
